terraform/custom_domains/environment_domains/terraform.tf

```diff
@@ -0,0 +1,25 @@
+terraform {
+
+  required_version = "= 1.4.5"
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "3.45.0"
+    }
+  }
+  backend "azurerm" {
+  }
+}
+
+provider "azurerm" {
+  features {}
+
+  skip_provider_registration = true
+}
+
+provider "azurerm" {
+  features {}
+  version         = "3.45.0"
+  alias           = "app_subcription"
+  subscription_id = var.app_sub_id # not in tra
+}
```

Comment:
It looks like we don't use this provider anywhere - can we remove it and the dependent variables?

Reply:
This has been removed